feg/protos/s6a_proxy.proto

```diff
@@ -168,7 +168,7 @@ message UpdateLocationAnswer {
 
     NetworkAccessMode network_access_mode = 8;
 
-    repeated string regional_subscription_zone_code = 9;
```

Comment:
Is it already being used by a deployed FeG? If yes, it may not be a safe change.